app/helpers/components_helper.rb

```diff
@@ -24,6 +24,10 @@ def mentor_inbox(conversations_request, tracks_request, sort_options)
                     })
   end
 
+  def mentor_solutions_list(request, sort_options)
+    react_component("mentor-solutions-list", { request: request, sort_options: sort_options })
```

Comment:
I feel like `sort_options` should be hard coded into this helper (and class when #82 is merged). I don't think it's the responsibility of the controller and the test controller to pass it it. The sort options are part of the actual component.

Reply:
I think the sort options should be passed in by the controller as a list of available options to sort by. This makes our client side code easier to change in cases where we want to remove a sorting option. This also keeps the list of sorting options in one place as we also need the sorting options on the server (to determine what option is selected, etc.)

Reply:
Yeah - I agree with that. But I think they should live in the component class not the controller class on the server.

Reply:
I'll PR it seperately.